policy-report/kep/README.md

```diff
@@ -173,6 +173,33 @@ updates.
 [documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
 -->
 
+This is a proposal to migrate the [Policy Report API](https://htmlpreview.github.io/?https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/docs/index.html) to a SIG repository.
+
+The Policy Report API was developed under the Kubernetes Policy Working Group and provides a standardized API that can be used by any policy engine, security scanner, or other security and compliance tool that wants to produce or consume policy results or findings for cluster resources.
+
+The Policy Report API is currently hosted at: https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report and is used by the following projects:
+
+Report producers:
+* [Kyverno](https://kyverno.io/docs/policy-reports/)
+* [jsPolicy](https://github.com/loft-sh/jspolicy/pull/17)
+* [Falco](https://github.com/falcosecurity/falcosidekick/blob/master/outputs/policyreport.go)
+* [Trivy Operator](https://aquasecurity.github.io/trivy-operator/v0.15.1/tutorials/integrations/policy-reporter/)
+* [Tracee Adapter](https://github.com/fjogeleit/tracee-polr-adapter)
+
+Report consumers:
+* [Kyverno Policy Reporter](https://kyverno.github.io/policy-reporter/)
+* [Open Cluster Management](https://open-cluster-management.io/)
+* [Lula](https://github.com/defenseunicorns/lula)
+
```

Comment:
Fairwinds Insights now consumes the policy report as well.

```suggestion
* [Fairwinds Insights](https://fairwinds.com/insights)

```

Reply:
Nice, thanks!